content/jobs.rst

```diff
@@ -47,14 +54,8 @@ You:
 - Work closely with designers to integrate form & function
 - Enjoy the push-and-pull of a tight feedback cycle
 - Enjoy exploring outside what you know, and questioning your assumptions
+- Are regularly available to check-in during US work hours,
+  roughly 10am-4pm Central Time
 
-We’re looking for 10-20 billable hours per week to start,
-with potential for more.
-@@@ Something tho express that "full time" is ~25hrs,
-this is not as half-time as it looks?
-
-Our process keeps design and development tightly integrated.
-There are no Photoshop comps thrown over the wall to developers.
-Everyone is working together (with clients and users) towards a unified goal.
 While we are specifically hiring for JavaScript experience,
 we’re interested in the full range of your skills and interests.
```

Comment:
Maybe also include our email address? Probably not a big deal -- we can funnel people through the contact form.